ui/00_message/skill/normal_skill_name_06.xml

```diff
@@ -2,17 +2,17 @@
   <message>
     <key>NORMAL_SKILL_NAME_06_92</key>
     <original>マジックボール</original>
-    <translation>Magic Balls</translation>
+    <translation>Magick Bolts</translation>
```

Comment:
It's a skill mirror from the other game. Bolts make sense if we have skill mirror names from DD:DA in here.